deepwell/src/services/page/service.rs

```diff
@@ -565,6 +565,28 @@ impl PageService {
         Ok(page)
     }
 
+    /// Gets all deleted pages that match the provided slug.
+    pub async fn get_deleted_slug(
```

Comment:
I would suggest this name:
```suggestion
    pub async fn get_deleted_by_slug(
```